runelite-client/src/main/java/net/runelite/client/plugins/nightmarezone/NightmareZonePlugin.java

```diff
@@ -73,18 +74,24 @@
 
 	@Getter
 	private int pointsPerHour;
-	
+
 	private Instant nmzSessionStartTime;
 
 	// This starts as true since you need to get
 	// above the threshold before sending notifications
 	private boolean absorptionNotificationSend = true;
+	private boolean overloadNotificationSend = false;
+	private Duration overloadWarningOffset;
+	private Instant lastOverload;
 
 	@Override
 	protected void startUp() throws Exception
 	{
 		overlayManager.add(overlay);
 		overlay.removeAbsorptionCounter();
+
+		updateWarningOffset();
+		lastOverload = Instant.now();
```

Comment:
Why is this not just `overloadNotificationSend = false`? It looks like this would break if you flipped the plugin off/on while overloaded currently.